### PR TITLE
Fix Broken Links

### PR DESCRIPTION
**Pull request by tom-frantz**
When I was reading through the website, I found a broken link in the maintainers section. I thought to fix it, and when I went to `CONTRIBUTING.md` to double check what to do, they were also broken!

### CODE REVIEW

File: templates/foundation.html [hunk 1]
```diff
@@ -72,7 +72,7 @@ <h2>Leadership</h2>
             <div>Secretary</div>
         </div>
     </div>
-    
+
```
Comment (tom-frantz) [inline templates/foundation.html:75]: These white space changes seem to be inline with the rest of the file

Comment (mockersf) [reply templates/foundation.html:75]: could you remove those extra changes? they are probably introduced by your IDE auto formatter

Comment (alice-i-cecile) [reply templates/foundation.html:75]:
```suggestion
    
```

File: templates/foundation.html [hunk 2]
```diff
@@ -83,7 +83,7 @@ <h2>Leadership</h2>
             <a href="https://github.com/mockersf">@mockersf</a>
         </div>
     </div>
-    
+
```
Comment (alice-i-cecile) [inline templates/foundation.html:86]:
```suggestion
    
```

File: templates/foundation.html [hunk 3]
```diff
@@ -94,7 +94,7 @@ <h2>Leadership</h2>
             <a href="https://github.com/superdump">@superdump</a>
         </div>
     </div>
-    
+
```
Comment (alice-i-cecile) [inline templates/foundation.html:97]:
```suggestion
    
```

File: templates/foundation.html [hunk 4]
```diff
@@ -115,7 +115,7 @@ <h2>Transparency</h2>
 
 <p>
 <b>Bevy Foundation</b> is a Washington state non-profit (with a pending 501(c)(3) application). We strive to make our operations transparent:
-</p> 
+</p>
```
Comment (alice-i-cecile) [inline templates/foundation.html:118]:
```suggestion
</p> 
```

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**